app/main/routes.py

```diff
@@ -381,12 +463,15 @@ def search():
         total_records = fuzzy_search_query.count()
         if total_records:
             num_records_found = total_records
+        else:
+            num_records_found = 0
```

Comment:
I know already discussed, but count will always return a number so no need to check if total records. could just do `num_record_found = fuzzy_search_query.count()` https://docs.sqlalchemy.org/en/14/orm/query.html#sqlalchemy.orm.Query.count

Reply:
I can do it in AYR-765 ticket if that works for you
![image](https://github.com/nationalarchives/da-ayr-beta-webapp/assets/107114665/12a5b745-ab9d-4422-948d-2d7a12157242)